multimodal/setup.py

```diff
@@ -25,15 +25,14 @@
     'Pillow',
     'tqdm',
     'boto3',
-    'setuptools<=59.5.0',
```

Comment:
@ddelange Trying to upgrade to PT 1.11 and remove the setuptools pin in this PR.